libvast/builtins/formats/json.cpp

```diff
@@ -516,6 +516,11 @@ struct config {
   }
 };
 
+auto add_common_options_to_parser(argument_parser& parser, config& cfg)
```

Comment:
A ndjson option will also be supported by a suricata/zeek-json and json in the next PR. This is why i have a function that handles all common options for all json alike parsers

Reply:
I think `suricata` and `zeek-json` should always be NDJSON. Their format is specified to be that, so there's little need to use something else as a default.

Reply:
Ok. I think the function is still fine to be left as it is. We may have more common options in the future.